src/modules/shared/layouts/AppLayout/AppLayout.tsx

```diff
@@ -162,6 +186,14 @@ const AppLayout: FC = ({ children }) => {
 		materialRequests && dispatch(setMaterialRequestCount(materialRequests?.items.length));
 	}, [dispatch, materialRequests]);
 
+	useEffect(() => {
+		if (!isLoggedIn) {
+			return;
+		}
+
+		getVisitorSpaces();
+	}, [getVisitorSpaces, isLoggedIn]);
```

Comment:
You should use the react-query hook:
```
useGetAllActiveVisits
```

You can use the enabled option to enable/disable the hook:
```
enabled: user && ![GroupName.KIOSK_VISITOR, GroupName.ANONYMOUS].includes(user.groupName)
```

and set de default sort to
```
			orderProp: 'endAt',
			orderDirection: OrderDirection.desc,
```